doc/tutorials/open_vswitch.rst

```diff
@@ -0,0 +1,163 @@
+.. _open_vswitch:
+
+Using OVS bridge instead of ACRN bridge
+#######################################
+This tutorial describes how to use Open Virtual Switch
+(OVS) bridge instead of ACRN bridge.
+
+.. note::
+   The OVS is integrated in ClearLinux ``service-os``
+   bundle, and the ClearLinux OS must be at least ``28900`` version.
+
+What is OVS
+***********
+Open vSwitch, abbreviated as OVS, is an open-source implementation of
+a distributed virtual multilayer switch which is used to provide a switching
+stack for hardware virtualization environments, while supporting multiple
+protocols and standards used in computer networks.
+
+Open vSwitch supports the following features:
+
+- Visibility into inter-VM communication via NetFlow,
```

Comment:
Yes, ACRN bridge does support the features .
The OVS does not replace ACRN bridge, we only integrate OVS in service-os bundle, and a guide for OVS setup, the users of OVS should be clear about the usage scenarios of OVS and enable OVS according to our guide.

Reply:
You say "OVS does not replace ACRN bridge", but the title of this article is "Using OVS bridge **instead of** ACRN bridge", and your comment in the intro says "customer can replace it with OVS by requirement"?

When I read this section, it sounds like OVS is better than ACRN bridge because OVS supports all these listed features that aren't supported by ACRN bridge.  Why are you listing all these features if ACRN bridge supports them too?  You do mention that "OVS can support 802.1Q feature to setup vlan network, and ACRN bridge can not meet this requirement." Are there other differences?

We should add something to the introduction (lines 5-8) to explain why someone might want to replace ACRN bridge with OVS.

Reply:
Agree with your point,  I want to change the title from "Using OVS bridge instead of ACRN bridge" to "How to enable OVS in ACRN" and remove all features introduction of OVS, since I don't want to indicate that OVS is better than ACRN brige, I just only want to introduce how to enable OVS in ACRN, and customer should know the difference between them.

what about your opinion?

Reply:
I think this is the right tone for this document, i.e.: explain how someone can use ACRN with OVS.

Is there any such thing as an "ACRN bridge" by the way? My (possibly limited) understanding was that these are all standard Linux mechanisms that we leverage and set-up correctly in our ACRN scripts and instructions. If true, I think a good intro would be to explain that by default, we provide scripts and configuration files that set-up such bridge (https://wiki.linuxfoundation.org/networking/bridge) and that this document explains how to use OVS instead (and "instead" is the right word here, it's one or the other).

I also believe it's useful to have a few top-level reasons mentionned in the intro as to why someone may want to use OVS. The point being that if the Linux bridge is enough, there is no reason to change it but if more advanced features are required then OVS may be a better solution for them. This page on the OVS website is dedicated to explaining why and when using OVS would make sense. I would recommend we refer to that document as well in our introduction.

Reply:
I agree with both of you.  Remove all the details about OVS but keep a simple overview. Changing the title makes sense too. @liuyuan-GitHub can you incorporate @gvancuts comments in a new version of this introduction.